assets/versions.js

```diff
@@ -17,6 +17,7 @@ function initializeVersionPlaceholder() {
 
 	for (version of versions) {
 		var active = documentPathname.startsWith("/" + version);
+		if (!active && documentPathname.startsWith("/latest") && version == versions[versions.length - 1]) active = true;
```

Comment:
This may be not correct when we publish non-final releases (beta, rc): such version is the last in the list of versions but it is not latest...

Reply:
Extracted latest version to var, so rc/beta versions in safe now.

*sigh one more thing to change on releases*